config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -4,20 +4,19 @@ presubmits:
     decorate: true
     skip_report: false
     run_if_changed: ".*/manifest.yaml"
-    # It's OK to run things concurrently, because we're using "-dry-run".
     max_concurrency: 10
     branches:
     - ^master$
     spec:
       containers:
-      - image: gcr.io/cip-demo-staging/cip:20190401-v1.0.0-3-g44d6169
+      - image: gcr.io/cip-demo-staging/cip:20190402-v1.0.2-0-g45293ca
         command:
         - multirun.sh
         args:
         - /app/cip-docker-image.binary
-        - k8s-staging-cluster-api/manifest.yaml
-        - k8s-staging-coredns/manifest.yaml
-        - k8s-staging-csi/manifest.yaml
+        - k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml
+        - k8s.gcr.io/k8s-staging-coredns/manifest.yaml
+        - k8s.gcr.io/k8s-staging-csi/manifest.yaml
```

Comment:
What's the rationale for moving these to gcr.io?

Reply:
@tpepper we are setting up repositories that teams can manage themselves. for example they can upload to these repos and then request these images to be uploaded to the main repo from there.

Reply:
As for the actual diff, I forgot that the yaml files in the repo need to be referenced by their relative path from the root of the git repo (in this case, the github.com/kubernetes/k8s.io repo). The older paths without the leading `k8s.gcr.io` suffix was plain wrong and the promoter complained about not being able to open those files... doh.